src/chunk.c

```diff
@@ -3207,32 +3218,21 @@ ts_chunk_do_drop_chunks(Hypertable *ht, int64 older_than, int64 newer_than, int3
 	{
 		int i;
 
-		/* Exclusively lock all chunks, and also refresh and invalidate the
+		/* Exclusively lock all chunks, and invalidate the
 		 * continuous aggregates in the regions covered by the chunks. Locking
 		 * prevents further modification of the dropped region during this
 		 * transaction, which allows moving the invalidation threshold without
 		 * having to worry about new invalidations while refreshing. */
 		for (i = 0; i < num_chunks; i++)
 		{
-			int64 start = chunk_primary_dimension_start(&chunks[i]);
-			int64 end = chunk_primary_dimension_end(&chunks[i]);
+			int64 start = ts_chunk_primary_dimension_start(&chunks[i]);
+			int64 end = ts_chunk_primary_dimension_end(&chunks[i]);
 
 			LockRelationOid(chunks[i].table_id, ExclusiveLock);
```

Comment:
I am not sure we strictly need to take this lock anymore. But I guess it doesn't hurt since it will probably be taken anyway once the chunk table is dropped.